rust/noosphere-storage/Cargo.toml

```diff
@@ -18,8 +18,8 @@ homepage = "https://github.com/subconsciousnetwork/noosphere"
 readme = "README.md"
 
 [features]
-default = ["kubo-storage"]
-kubo-storage = ["reqwest"]
+default = ["ipfs-storage"]
+ipfs-storage = ["noosphere-ipfs"]
```

Comment:
s/kubo-storage/ipfs-storage where appropriate as the IPFS HTTP Gateway is at least implemented by Iroh (and "standardized")